### PR TITLE
Dk avdpoc 20231101

### PR DESCRIPTION
**Pull request by dkeitaarista**

### CODE REVIEW

File: .github/workflows/feature_branches.yml
Comment (dkeitaarista) [inline .github/workflows/feature_branches.yml:1]: accepted

File: .github/workflows/main_branch.yml
Comment (dkeitaarista) [inline .github/workflows/main_branch.yml:1]: accepted

File: .pre-commit-config.yaml
Comment (dkeitaarista) [inline .pre-commit-config.yaml:1]: ok

File: Makefile
Comment (dkeitaarista) [inline Makefile:1]: ok

File: edj-act-topo-dk.yml
Comment (dkeitaarista) [inline edj-act-topo-dk.yml:1]: ok

File: inventory/.DS_Store
Comment (dkeitaarista) [inline inventory/.DS_Store:1]: ok

File: inventory/MPLS_network/.DS_Store
Comment (dkeitaarista) [inline inventory/MPLS_network/.DS_Store:1]: ok

File: inventory/MPLS_network/documentation/devices/BRANCH-A2A-CE1.md
Comment (dkeitaarista) [inline inventory/MPLS_network/documentation/devices/BRANCH-A2A-CE1.md:1]: ok

**Review by dkeitaarista — COMMENTED** [review]
Reviewed

**Review by keitadam — APPROVED** [review]
Approved

**Comment by keitadam** [comment]
Approval completed

**Pull request closed without merging**